src/pill/pill.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { PillProps } from "./types";
+import { Content, Icon, Label, Wrapper } from "./pill.style";
+
+export const Pill = ({
+    type,
+    colorType = "black",
+    className,
+    children,
+    icon,
+    ...otherProps
+}: PillProps): JSX.Element => {
+    return (
+        <Wrapper
+            className={className}
+            data-testid={otherProps["data-testid"]}
+            $type={type}
+            $color={colorType}
+        >
+            <Content>
```

Comment:
this extra nesting with `Content` seems redundant, can it be merged with `Wrapper`?